z02-parts-of-a-graph.md

```diff
@@ -155,8 +157,8 @@ d3.extent(numbers);
 
 D3 has objects called scales that help map values across coordinate systems.
 There are different kinds of scales (linear, logarithmic, linear for time).
-Scales are configured with a domain and a range, they map from the data to
-the approprate part of the screen (screen space).
+Scales are configured with a Domain (the raw data) and a Range (pixel plot size),
```

Comment:
Let's not capitalize "Domain" and "Range" here, they're not proper nouns.

Would this sentence reflect the clarification you're trying to make?

> Scale are configured with a domain (the raw data) and a range (the pixels to output to). As functions, they map this raw data to the appropriate part of the screen (screen space).